### ISSUE TITLE
Varia Child Themes: Menu Items Need Clearer Labelling for Screen Readers

### ISSUE THREAD

**Issue by donalirl** [Automattic/themes#2000]
## Steps to replicate

1. Navigate to the primary menu of any Varia child theme with a screen reader.
2. Swiping left to right and the screen reader picks up a bunch of unnecessary and poorly labelled items. For example, Varia reads them off in this order:

* Menu expanded collapsed pop up button
* menu
* plus
* expanded
* collapsed

Recreated on Varia parent theme and Rockfield child theme.

## Screencast

https://d.pr/v/1BP4zh (from Rockfield)

## Result

Visually impaired users using a screen reader cannot easily identify how to navigate to the menu. In addition, it takes a long time to reach the site content because of the extraneous items.

## Expected

I expected to be able to easily navigate to a menu, through a menu, and past a menu to the rest of the content while using a screen reader.

* User report: #2889143-zen #p4vt7e-6n-p2
